docs/adr/template.md

```diff
@@ -0,0 +1,73 @@
+# [short title of solved problem and solution]
+
+* Status: [proposed | rejected | accepted | deprecated | … | superseded by [ADR-0005](0005-example.md)] <!-- optional -->
+* Deciders: [list everyone involved in the decision] <!-- optional -->
```

Comment:
I think people who proposed the AD are more relevant here. Since people who proposed the AD are responsible for pushing for it, responding to feedback and making sure the changes are sound.

```suggestion
* Authors: [list everyone who authored the AD]
```

Do we need sections for people who approved the AD or offered suggestions?

Reply:
I agree, I think authors are who should be listed and they are easier to determine.